Dolapo47/Auto-Mart
# create new car ad route

#### What does this PR do?
- A summary of what this task is about
this PR creates a car route for the app
#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
-imported the necessary dependencies
-setup cloudinary and multer for file upload
-destructured the request body
-query the database
-pass a response to user
#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
run npm start on local machine
fill in ithe input fields
send the request
#### What are the relevant pivotal tracker stories?
- PT Story link - (format - `[story_id](https://www.pivotaltracker.com/n/projects/2346179)`)
